src/main/pg/motor.c

```diff
@@ -67,9 +67,22 @@ void pgResetFn_motorConfig(motorConfig_t *motorConfig)
 #endif
 
 #ifdef USE_TIMER
-    for (int motorIndex = 0; motorIndex < MAX_SUPPORTED_MOTORS; motorIndex++) {
-        motorConfig->dev.ioTags[motorIndex] = timerioTagGetByUsage(TIM_USE_MOTOR, motorIndex);
-    }
+#ifdef MOTOR1_PIN
+    motorConfig->dev.ioTags[0] = IO_TAG(MOTOR1_PIN);
+#endif
+#ifdef MOTOR2_PIN
+    motorConfig->dev.ioTags[1] = IO_TAG(MOTOR2_PIN);
+#endif
+#ifdef MOTOR3_PIN
+    motorConfig->dev.ioTags[2] = IO_TAG(MOTOR3_PIN);
+#endif
+#ifdef MOTOR4_PIN
+    motorConfig->dev.ioTags[3] = IO_TAG(MOTOR4_PIN);
+#endif
```

Comment:
Would this be better implemented using a macro?

Reply:
Could do it similar to timer pin mapping, ie motor pin mapping?